ddp-workspace/projects/ddp-dsm-ui/src/app/filter-column/filter-column.model.ts

```diff
@@ -239,6 +242,26 @@ export class Filter {
     new NameValue('review', 'Second Abstraction'),
     new NameValue('qc', 'QC') ]);
   public static ABSTRACTION_USER = new Filter(ParticipantColumn.ABSTRACTION_USER, Filter.TEXT_TYPE);
+  public static ACTIVITY_STATUS = new Filter(ParticipantColumn.ACTIVITY_STATUS, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null,
+    (participant: Participant,  activityDefinitionList: ActivityDefinition[])=>{
+      let str = "";
+      let activityDataArray = participant.data.activities;
```

Comment:
why do we need participant.data.activities here. do we not have all info in activityDefinitionList?

Reply:
we want to sort activities in the order that was in their definition

Reply:
yes, but that order is in activity definition. why do we need participant object?

Reply:
basically here I am sorting that participant's activities based on their order, which is in the definition. So I would need the participant to access their activities, and the definition to sort them